### DISCUSSION TITLE
MAUI example of implementation with SiriKit and Google Assistant

### DISCUSSION THREAD

**Discussion by eeadie** [dotnet/maui#24928] [Q&A]
Hi,

Is there any documentation/examples of an implementation with SiriKit and Google Assistant for MAUI? I have found an old Xamarin article but can't find anything for MAUI:

[Implementing SiriKit in Xamarin.iOS](https://learn.microsoft.com/en-us/previous-versions/xamarin/ios/platform/sirikit/implementing-sirikit?tabs=macos%22https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fxamarin%2Fios%2Fplatform%2Fsirikit%2Fimplementing-sirikit%3Ftabs%3Dmacos%22)

Thanks,

Euan

**Comment by Hurricane31337** [comment]
I second this. Through the rise of AI, our apps need to support these smart ways of interacting. Like "Hey Siri, navigate to my next appointment in [our AppName]."

**Comment by Hurricane31337** [comment]
Any update on this?
